### ISSUE TITLE
ValueError handling classifiers as tuple instead of list

### ISSUE THREAD

**Issue by mortenlj** [domenkozar/g-pypi#12]
```
root@host # gpypi create dotfiles
 * Generating ebuild: dotfiles 0.6.0
Traceback (most recent call last):
  File "/usr/bin/gpypi-2.7", line 9, in <module>
    load_entry_point('g-pypi==0.3', 'console_scripts', 'gpypi')()
  File "/usr/lib64/python2.7/site-packages/gpypi/cli.py", line 410, in main
    CLI(config_mgr)
  File "/usr/lib64/python2.7/site-packages/gpypi/cli.py", line 208, in __init__
    getattr(self, config.command)()
  File "/usr/lib64/python2.7/site-packages/gpypi/cli.py", line 215, in create
    gpypi.create_ebuilds()
  File "/usr/lib64/python2.7/site-packages/gpypi/cli.py", line 64, in create_ebuilds
    requires = self.do_ebuild()
  File "/usr/lib64/python2.7/site-packages/gpypi/cli.py", line 175, in do_ebuild
    ebuild.create()
  File "/usr/lib64/python2.7/site-packages/gpypi/ebuild.py", line 462, in create
    self.post_unpack()
  File "/usr/lib64/python2.7/site-packages/gpypi/ebuild.py", line 268, in post_unpack
    metadata = Enamer.parse_setup_py(d)
  File "/usr/lib64/python2.7/site-packages/gpypi/enamer.py", line 188, in parse_setup_py
    d['license'] = cls.convert_license(distribution.get_classifiers(), distribution.metadata.get_license())
  File "/usr/lib64/python2.7/site-packages/gpypi/enamer.py", line 604, in convert_license
    raise ValueError("classifiers should be a list, not %s" % type(classifiers))
ValueError: classifiers should be a list, not <type 'tuple'>
root@host # gpypi --version
gpypi 0.3
```

**Comment by mortenlj** [comment]
Removing setup_py from the "use" setting in /etc/gpypi allows me to get past this, but reading information from setup.py is very useful.

**Comment by sjlongland** [comment]
I found I was able to kludge around this by adding the following right above line 604 (where it tests for `isinstance(..., list)`)

```
try:
    classifiers = list(classifiers)
except:
    pass
```

The other way might be to amend the if statement to also check for a tuple.  I'm not sure if it actually needs the list semantics or whether it just needs something _iterable_, and a list happens to be one of a few iterable containers.

Another one that (indirectly) triggers it is socketIO-client: one of its dependencies triggers this bug in gpypi.
